inc/commonitilobject.class.php

```diff
@@ -6866,7 +6866,21 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
     *
     * @return mixed[] Timeline items
     */
-   function getTimelineItems($from_notification = false, $get_private = false) {
+   function getTimelineItems($options) {
```

Comment:
You should also update PHPdoc.
```suggestion
   function getTimelineItems(array $options = []) {
```

Reply:
done